dipy/reconst/dki.py

```diff
@@ -28,6 +28,459 @@
 from .base import ReconstModel
 
 
+def _positive_evals(L1, L2, L3, er=None):
+    """ Helper function that indentifies which voxels in a array have all
+    eigenvalues larger than zero
+
+    Parameters
+    ----------
+    L1 : ndarray (n,)
```

Comment:
I think this would work on ndarrays regardless of their shape. This seems to imply that it would only work with 1D arrays.